src/main/java/cc/polyfrost/oneconfig/renderer/TinyFD.java

```diff
@@ -1,4 +1,70 @@
 package cc.polyfrost.oneconfig.renderer;
 
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.io.File;
+
+@SuppressWarnings("unused")
 public interface TinyFD {
-}
+    String QUESTION_ICON = "question";
+    String ERROR_ICON = "error";
+    String WARNING_ICON = "warning";
+    String INFO_ICON = "info";
+
+    String OK_DIALOG = "ok";
+    String OK_CANCEL_DIALOG = "okcancel";
+    String YES_NO_DIALOG = "yesno";
+    String YES_NO_CANCEL_DIALOG = "yesnocancel";
```

Comment:
i18n when

Reply:
this is just TinyFD stuff iirc

Reply:
> this is just TinyFD stuff iirc

ah okay, does tinyfd expose these constants?

Reply:
ye